xlgui/widgets/dialogs.py

```diff
@@ -139,6 +139,35 @@ def on_response(self, *_):
         self.destroy()
 
 
+@GtkTemplate('ui', 'shortcuts_dialog.ui')
+class ShortcutsDialog(Gtk.Dialog):
+    '''
+        Shows information about registered shortcuts
+        
+        TODO: someday upgrade to Gtk.ShortcutsWindow when we require 3.20 as
+              a minimum GTK version
+    '''
+    
+    # doesn't work if we don't set the treeview here too..
+    shortcuts_treeview, shortcuts_model = GtkTemplate.Child.widgets(2)
+    
+    __gtype_name__ = 'ShortcutsDialog'
+    
+    def __init__(self, parent=None):
+        Gtk.Dialog.__init__(self)
+        self.init_template()
+        
+        self.set_transient_for(parent)
+        
+        for a in sorted(providers.get('mainwindow-accelerators'), key=lambda a: a.name[-1] + a.name):
```

Comment:
Suggestion: `key=lambda a: (a.key, a.mods)` so it works for things like `F1`, `Delete`, etc.